src/core/ui.js

```diff
@@ -195,6 +234,7 @@ export const ui = {
     overlay.addEventListener("click", () => this.closeModal(currentOwner));
     overlay.classList.toggle("respec-show-overlay");
     modal.hidden = false;
+    trapFocus(modal);
```

Comment:
I suggest we defer this, as:

1. The focus does not enter the modal window, so not trapped. This one is easy to fix.
2. Trapping doesn't work when the modal window dynamically modifies its tree (specref) or uses iframe (definition search). This one is harder.

This PR is for the pill, I would defer anything irrelevant for future new PRs.

Reply:
Okay! I am putting on hold and start working on the two above points you just mentioned :)

Reply:
```suggestion
```

Let's do the modal thing in the next PRs and complete this one 🎉